frontend/src/Playground/index.js

```diff
@@ -6,22 +6,52 @@
  * license that can be found in the LICENSE file.
  */
 
-import React, { useState, useEffect } from 'react';
-import './Playground.css';
 import { Client } from '@rundsk/js-sdk';
+import React, { useEffect, useState } from 'react';
+import ReactDOMServer from 'react-dom/server';
+import { useGlobal } from 'reactn';
+import './Playground.css';
 
 function Playground(props) {
+  const [source] = useGlobal('source');
+
+  const [iframeSourceURL, setIframeSourceURL] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+
   const [annotationData, setAnnotationData] = useState({ annotations: [] });
   const [highlightedAnnotation, setHighlightedAnnotation] = useState(null);
 
   if (props.src) {
-    console.debug('<Playground> props.src has been deprecated in favor of props.annotate.');
+    console.warn('<Playground> props.src has been deprecated in favor of props.annotate.');
     props.annotate = props.src;
     delete props.src;
   }
 
+  useEffect(() => {
+    Client.playgroundURL(props.node.url, props.doc.id, props['data-component'], source).then(setIframeSourceURL);
+  }, [setIframeSourceURL, props.node.url, props.doc.id, props.children, source]);
```

Comment:
Would we want `props['data-component']` in the dependency array?

Reply:
Good catch. Yes, that was an oversight.